service/lib/agama/cmdline_args.rb

```diff
@@ -68,6 +70,13 @@ def self.read_from(path)
       args
     end
 
+    # Despite Agama is young it already contains some relicts. This method should purge them and put
+    # command line options into a standard shape
+    def self.standardize(option)
+      # agama. is now obsolete original kernel argument prefix
+      option.sub("agama.", CMDLINE_PREFIX)
```

Comment:
sadly this does not work as you intend as it replace also inside of values. So it is quite fragile:
```
jreidinger@localhost:~/prace> irb
irb(main):001:0> "my_arg=agama.place".sub("agama.", "inst.")
=> "my_arg=inst.place"
```

Reply:
similarly if you use e.g. `inst.info=http://agama.host.com/my.info` it will be broken.

Reply:
it works ... because next line filters such key, value pairs out (it requires option to start_with CMDLINE_PREFIX)

but i agree it is not good practise and will polish it